lttng-tools: Add support for v4l2 probes

Support insertion of this kernel module on start of the session.

Signed-off-by: Wade Farnsworth <wade_farnsworth@mentor.com>

Index: git/src/bin/lttng-sessiond/modprobe.c
===================================================================
--- git.orig/src/bin/lttng-sessiond/modprobe.c	2013-05-15 10:55:40.892586624 -0700
+++ git/src/bin/lttng-sessiond/modprobe.c	2013-05-22 12:03:53.172607690 -0700
@@ -80,6 +80,7 @@ const struct kern_modules_param kern_modules_list[] = {
 	{ "lttng-probe-vmscan", 0 },
 	{ "lttng-probe-workqueue", 0 },
 	{ "lttng-probe-writeback", 0 },
+	{ "lttng-probe-v4l2", 0 },
 };
 
 /*
